plugins/source/azure/client/services/resources.go

```diff
@@ -11,41 +11,33 @@ import (
 )
 
 type ResourcesClient struct {
-	Groups      GroupsClient
-	Resources   ResClient
-	Assignments AssignmentsClient
-	Links       LinksClient
+	Groups            ResourcesGroupsClient
+	PolicyAssignments ResourcesPolicyAssignmentsClient
+	Links             ResourcesLinksClient
 }
 
-type GroupsClient interface {
+type ResourcesGroupsClient interface {
 	List(ctx context.Context, filter string, top *int32) (result resources.GroupListResultPage, err error)
 }
 
-type ResClient interface {
-	List(ctx context.Context, filter string, expand string, top *int32) (result resources.ListResultPage, err error)
-}
-
-type AssignmentsClient interface {
+type ResourcesPolicyAssignmentsClient interface {
 	List(ctx context.Context, subscriptionID string, filter string, top *int32) (result policy.AssignmentListResultPage, err error)
 }
 
-type LinksClient interface {
+type ResourcesLinksClient interface {
 	ListAtSubscription(ctx context.Context, filter string) (result links.ResourceLinkResultPage, err error)
 }
 
 func NewResourcesClient(subscriptionId string, auth autorest.Authorizer) ResourcesClient {
 	groups := resources.NewGroupsClient(subscriptionId)
 	groups.Authorizer = auth
-	client := resources.NewClient(subscriptionId)
-	client.Authorizer = auth
 	assignments := policy.NewAssignmentsClient()
 	assignments.Authorizer = auth
 	ls := links.NewResourceLinksClient(subscriptionId)
 	ls.Authorizer = auth
 	return ResourcesClient{
-		Groups:      groups,
-		Resources:   client,
```

Comment:
`Resources` moved to `plugins/source/azure/client/services/monitor.go` as it's not used to get anything under the `resources` package